### PR TITLE
MS-756: PDF saveValues- fix unit tests

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
* Make functions use async & await feature of node 8
* Make tests use async & await feature of node 8
* Make async functions synchronous
* Makes the functions easier to read and write
* Makes it less messy

### CODE REVIEW

File: apps/pdf/behaviours/generate-send-pdf.js [hunk 1]
```diff
@@ -38,21 +38,20 @@ const deleteFile = (file) => {
...
-      })
-      .then(file => {
-        deleteFile(file);
-      })
-      .catch(err=> console.log(`pdf generation error -> ${err}`));
 
-    super.saveValues(req, res, (err) => {
-      next(err);
-    });
+    // Try catch for async function because PDFgenerate function returns a promise need to manage any errors.
```
Comment (jjrobothamho) [inline apps/pdf/behaviours/generate-send-pdf.js]: life's easier with async await nice.

**Comment by jjrobothamho** [comment]
well done

File: apps/pdf/behaviours/generate-send-pdf.js (hunk 1 above)
Comment (sulthan-ahmed) [reply apps/pdf/behaviours/generate-send-pdf.js]: i totally agree

**Comment by sulthan-ahmed** [comment]
Thanks

**Pull request merged**